app-libs/parentchain-interface/src/integritee/api_client_types.rs

```diff
@@ -37,14 +37,11 @@ pub type IntegriteeTip = PlainTip<Balance>;
 pub type IntegriteeExtrinsicParams = GenericExtrinsicParams<IntegriteeRuntimeConfig, IntegriteeTip>;
 pub type IntegriteeAdditionalParams = GenericAdditionalParams<IntegriteeRuntimeConfig, Hash>;
 
-pub type IntegriteeSignedExtra = GenericSignedExtra<IntegriteeTip, Index>;
-pub type IntegriteeSignature = Signature<IntegriteeSignedExtra>;
+pub type IntegriteeSignedExtra = GenericTxExtension<IntegriteeTip, Index>;
+pub type IntegriteeSignature = MultiSignature;
 
 pub type IntegriteeUncheckedExtrinsic<Call> =
-	UncheckedExtrinsicV4<Address, Call, PairSignature, IntegriteeSignedExtra>;
-
-/// Signature type of the [UncheckedExtrinsicV4].
-pub type Signature<SignedExtra> = Option<(Address, PairSignature, SignedExtra)>;
+	UncheckedExtrinsic<Address, Call, PairSignature, IntegriteeSignedExtra>;
 
```

Comment:
This was the tricky part as to why the node failed to interpret the extrinsic correctly. The signature used by the api client is now a plain `MultiSignature`